doc/tracepath.xml

```diff
@@ -162,28 +162,28 @@ root@mops:~ # tracepath -6 3ffe:2400:0:109::2
  2:  3ffe:2400:0:109::2               463.514ms reached
      Resume: pmtu 1480 hops 2 back 2
     </literallayout>
-    <para>The first column shows TTL of the probe, followed by
-    colon. Usually value of TTL is obtained from reply from
-    network, but sometimes reply does not contain necessary
+    <para>The first column shows the TTL of the probe, followed by
+    colon. Usually the value of TTL is obtained from the reply from
+    the network. But sometimes it does not contain the necessary
     information and we have to guess it. In this case the number is
     followed by ?.</para>
     <para>The second column shows the network hop, which replied to
-    the probe. It is either address of router or word [LOCALHOST],
-    if the probe was not sent to the network.</para>
+    the probe. It is either the address of the router or the word
+    [LOCALHOST], if the probe was not sent to the network.</para>
     <para>The rest of line shows miscellaneous information about
-    path to the corresponding network hop. As rule it contains
-    value of RTT. Additionally, it can show Path MTU, when it
+    the path to the corresponding network hop. At least it contains
+    the value of RTT. Additionally, it can show Path MTU, when it
```

Comment:
I'm not sure by some of the definite articles (`the`), but it's probably correct. A review from native speaker would be great.